projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployer.java

```diff
@@ -51,6 +51,8 @@ public class JobImageDeployer {
   private static final String KEYTAB_FILENAME_ENV = "VDK_KEYTAB_FILENAME";
   private static final String ATTEMPT_ID = "VDK_ATTEMPT_ID";
   private static final String BASE_CONFIG_FOLDER = "VDK_BASE_CONFIG_FOLDER";
+  private static final String VDK_TEMPORARY_WRITE_DIRECTORY = "VDK_TEMPORARY_WRITE_DIRECTORY";
+  private static final String TEMPORARY_WRITE_DIRECTORY_PATH = "/var/tmp";
```

Comment:
Hmm, Isn't TEMPORARY_WRITE_DIRECTORY_PATH actually "EPHEMERAL_VOLUME_MOUNT_PATH" ?

Do we need separate variable ? Or if we want for clarity sake we can just assign it 
`TEMPORARY_WRITE_DIRECTORY_PATH=EPHEMERAL_VOLUME_MOUNT_PATH`

Reply:
Yes, it is the same.

Reply:
Is there a situation where those two can be different folders? (that's why I decided to use a separate variable.) In any case I think we should have two different variables here.